src/model/user.ts

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose';
+const UserSchema = new mongoose.Schema(
```

Comment:
try to add prettier in your code to make readable

Reply:
thanks for feedback danilo